prime-router/src/test/kotlin/fhirengine/engine/FhirConverterTests.kt

```diff
@@ -612,8 +612,8 @@ class FhirConverterTests {
             every { mockMessage.topic } returns Topic.FULL_ELR
```

Comment:
Maybe update these test names since they say "and not return a bundle" but that's not being checked for in the tests any more/we are expecting them to return something now.